src/BaseModule.php

```diff
@@ -173,29 +173,41 @@ protected function put(array $request = [])
 	{
 	}
 
+	/**
+	 * Module OPTIONS method to process submitted data
+	 *
+	 * Extend this method if the module is supposed to process OPTIONS requests.
+	 * Doesn't display any content
+	 *
+	 * @param string[] $request The $_REQUEST content
+	 */
+	protected function options(array $request = [])
```

Comment:
Is this still used? I thought that there is now a special handling for the options?